src/collectors/nvidia_gpu/nvidia_gpu.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+
+"""
+The NvidiaGPUCollector collects GPU utilization metric using nvidia-smi.
+
+#### Dependencies
+
+ * /usr/bin/nvidia-smi
+
+"""
+
+from itertools import izip
+
+import diamond.collector
+
+
+class NvidiaGPUCollector(diamond.collector.ProcessCollector):
+
+    def get_default_config_help(self):
+        config_help = super(NvidiaGPUCollector, self).get_default_config_help()
+        config_help.update({
+            'bin': 'The path to the ping binary'
+        })
+        return config_help
+
+    def get_default_config(self):
+        """
+        Returns the default collector settings
+        """
+        config = super(NvidiaGPUCollector, self).get_default_config()
+        config.update({
+            'path': 'nvidia',
+            'bin': '/usr/bin/nvidia-smi',
+            'stats': [
```

Comment:
I would prefer to use `nvidia-smi` command line tool. Because Nvidia driver provides `nvidia-smi` tool and I don't have to install extra dependency.

Also the adoption of the library is really low, I am hesitate to use that library.